src/utils/Constants.tsx

```diff
@@ -21,7 +21,19 @@ export const ICONS: IconType = {
   reports: <SummarizeOutlined />,
 }
 
-export const NAV_ITEMS = [
+export interface NavItemsType{
+  icon:string;
+  text:string;
+  route:string;
+  nestedItems:NestedNavItemsType[];
```

Comment:
keep nestedItems type also as "NavItemsType"
Assuming if we have further sub items